### ISSUE TITLE
Set this app as deprecated

### ISSUE THREAD

**Issue by SerhiDi** [Shopify/product-reviews-sample-app#64]
### Issue

Please explain to me why this app and the app built from the Shopify CLI have different structures?
Why is this application not updated to have the updated structure?

For example, You are new to Shopify apps development. And you want to get some dynamic data from your shop. You will use `Shopify Node app`.
If you read Shopify documentation and want to create an app, you run the CLI command. After that, if you want to create an extension, you run the CLI command.

BUT, if you want to make some `fetch` from that extension, you will see the link to this repository `[product-reviews-sample-app](https://github.com/Shopify/product-reviews-sample-app)`. And the different app structure. What? Why?

This app uses `koa-router`. And the example `fetch` from the extension not working in the new application.
CLI app uses `express`.

And the app extensions documentation does not contain an example, it sends to this repository. Facepalm. 
That kind of attitude is just annoying.

**Comment by dwight9339** [comment]
I agree with @SerhiDi. Spent a few weeks building an app using this one as a model before realizing it needed some serious updates.
